.github/workflows/aiohttp.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+name: Aiohttp
+
+
+on:
+  merge_group:
+  push:
+    branches:
+    - master
+    tags:
+    - v*
+  pull_request:
+    branches:
+    - master
+
+
+env:
+  COLOR: "yes"
+  FORCE_COLOR: 1
+  PY_COLORS: 1
+
+
+jobs:
+
+  test-aiohttp:
+    name: Aiohttp tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - name: Checkout aiohttp
+      uses: actions/checkout@v4
+      with:
+        repository: aio-libs/aiohttp
+        submodules: true
+    - name: Checkout yarl
+      uses: actions/checkout@v4
+      with:
+        path: vendor/yarl
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.x
+        cache: pip
+        cache-dependency-path: requirements/*.txt
+    - name: Update pip, wheel, setuptools, build, twine
+      run: |
+        python -m pip install -U pip wheel setuptools build twine
```

Comment:
Nothing short of `build` seems to be necessary here.

Reply:
> Nothing short of `build` seems to be necessary here.

I just realized that we can remove this entire step, since the neccesary upgrades are already ensured by the makefile command.